app/clients/sms/aws_pinpoint.py

```diff
@@ -21,16 +21,21 @@ def init_app(self, current_app, statsd_client, *args, **kwargs):
     def get_name(self):
         return self.name
 
-    def send_sms(self, to, content, reference, multi=True, sender=None, template_id=None, service_id=None):
+    def send_sms(self, to, content, reference, multi=True, sender=None, template_id=None, service_id=None, sending_vehicle=None):
         messageType = "TRANSACTIONAL"
         matched = False
         opted_out = False
         response = {}
 
-        use_shortcode_pool = (
-            str(template_id) in self.current_app.config["AWS_PINPOINT_SC_TEMPLATE_IDS"]
-            or str(service_id) == self.current_app.config["NOTIFY_SERVICE_ID"]
-        )
+        if self.current_app.config["FF_TEMPLATE_CATEGORY"]:
+            use_shortcode_pool = (
+                sending_vehicle == "short_code" or str(service_id) == self.current_app.config["NOTIFY_SERVICE_ID"]
```

Comment:
Would it be possible to get the `sending_vehicle` options into an `enum` or something similar?

Reply:
That's a good idea! I've added an enum
```
class SmsSendingVehicles(Enum):
    SHORT_CODE = "short_code"
    LONG_CODE = "long_code"
```
And used it everywhere we use the sending vehicle (including in the database model)